ci/setup-db.sh

```diff
@@ -0,0 +1,41 @@
+#/usr/bin/env bash
+set -e
+
+if [ "$RUNNER_OS" == "Linux" ];
+echo "Install postgres"
+yum install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm || true
+yum install -y postgresql11-devel || true
+
+ln -sf /usr/pgsql-11/bin/pg_config /usr/bin/
+export RUNNER_TEMP="/tmp"
+fi
+
+echo "Set up database"
+
+export PGDATA="$RUNNER_TEMP/pgdata"
+export PWFILE="$RUNNER_TEMP/pwfile"
+
+echo 'testpwd' > $PWFILE
```

Comment:
You can probably set up the test role without password? To simplify the script. https://www.postgresql.org/docs/current/auth-trust.html

Reply:
Love it. 

Maybe let's just use the service on linux?

Reply:
Only win fails `FATAL:  role "postgres" does not exist`